FakeItEasy.Analyzers.sln

```diff
@@ -9,17 +9,17 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		.gitignore = .gitignore
 		.mailmap = .mailmap
 		build.cmd = build.cmd
-		build.sh = build.sh
 		build.ps1 = build.ps1
-		ci.yml
+		build.sh = build.sh
+		.github\workflows\ci.yml = .github\workflows\ci.yml

```

Comment:
the workflow files will show up right under "Solution Items". If you prefer a folder, I am happy to create and nest. I don't love a ".github/workflows" folder, so would suggest "workflows", but am open to whatever.

Reply:
> the workflow files will show up right under "Solution Items".

👍